app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_post_likes_counter
+
+  private
+
+  def update_post_likes_counter
+    post&.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] To meet the below requirement please add unit tests for this custom method created for the like model :+1: 
![mm](https://github.com/NoelLincoln/blog-app/assets/89055502/1a7065fb-a00b-41cf-b3e5-3f72e0e2a85e)